packages/replayio/src/commands/setup.ts

```diff
@@ -0,0 +1,100 @@
+import { existsSync } from "fs";
+import path from "path";
+import chalk from "chalk";
+import { registerCommand } from "../utils/commander";
+import { exitProcess } from "../utils/exitProcess";
+import {
+  findPlaywrightConfig,
+  installReplayDependencies,
+  setupReplayInPlaywrightConfig,
+} from "../utils/config/playwright";
+import { Command } from "commander";
+
+const setupCommand = registerCommand("setup").description("Setup Replay for test runner");
+const playwrightCommand = new Command("playwright")
+  .option("--config <path>", "Path to playwright config file")
+  .description("Setup Replay for Playwright test runner")
+  .action(setupPlaywright);
+setupCommand.addCommand(playwrightCommand);
+
+async function setupPlaywright({ config }: { config?: string }) {
+  const hasPackageJson = existsSync(path.join(process.cwd(), "package.json"));
+
+  if (!hasPackageJson) {
+    console.error(chalk.red("Please ensure you're in the Node.js project directory."));
+    await exitProcess(1);
```

Comment:
people often (-ish) run commands in subdirectories (they might just not be aware that they have changed their location for some reason). It would be great if this could work in those scenarios - to fix this you could recursively check parent directories to see if they have `package.json` file in them. A common pkg used for this is [`read-package-up`](https://github.com/sindresorhus/read-package-up) (or [`find-up-simple`](https://github.com/sindresorhus/find-up-simple)/[`find-up`](https://github.com/sindresorhus/find-up))